.gitignore

```diff
@@ -4,6 +4,7 @@
 
 ## Xcode Files
 project.xcworkspace
+xcshareddata/
```

Comment:
`xcsharedata` is for shared settings of the project right? I am not sure if this should be ignored in commits. Can you recheck if this is correct?

Reply:
Good catch! As [seen here](https://stackoverflow.com/questions/53034303/should-xcshareddata-be-checked-into-git-repo-for-xcode-project), I was right to add in `xcuserdata`, but not `xcsharedata`.

Will do a commit shortly :)

Reply:
[ac8fc7a](https://github.com/scribe-org/Scribe-iOS/pull/209/commits/ac8fc7a4975687896fce19757571a7c61b84570c) adds them in @SaurabhJamadagni 😊